classes/v1migration/v1migration.php

```diff
@@ -587,31 +649,28 @@ public static function turnitintooltwo_getassignments() {
     }
 
     /**
-     * Delete a list of assignments.
+     * Delete an assignment.
      *
-     * @param array $assignmentids The assignment IDs to delete.
+     * @param int $assignmentid The assignment ID to delete.
      */
-    public static function turnitintooltwo_delete_assignments($assignmentids) {
+    public static function turnitintooltwo_delete_assignment($assignmentid) {
```

Comment:
Was this only used by the migration tool? Don't want to break something somewhere else

Reply:
It is yeah. It calls `turnitintool_delete_instance()` as part of the function so nothing else will find that useful. Saying that I'm going to rename the function as its job is to delete a V1 assignment and the `turnitintooltwo` prefix could be confusing.